.travis.yml

```diff
@@ -24,7 +24,7 @@ install:
 # Python
 - pip freeze
 - pip install -r monkey/monkey_island/requirements.txt  # for unit tests
-- pip install flake8 pytest pytest-cov dlint isort # for next stages
+- pip install black==20.8b1 flake8==3.9.0 pytest pytest-cov dlint isort==5.8.0 # for next stages
```

Comment:
Make sure we use the same in our requirement file for island

Reply:
pre-commit uses the `.pre-commit-config.yaml` to track these requirements. I believe it installs them in a virtualenv separate from the rest of the python libraries on the system.